### PR TITLE
Update to 2015 graph - Fixes #520

### LINKED ISSUE #520: 2015-2016 Calendar Update
[The CSC graph needs to be updated with the 2015-2016 changes.](http://www.artsandscience.utoronto.ca/ofr/calendar/crs_csc.htm)

List of potential changes:
- [x] CSC466 needs to be added to the graph
- [x] CSC485 now has CSC207/209 instead of just 209
- [ ] Grade prerequisites for 207/236/258
- [x] CSC448H1 has only 236 for prereq now
- [x] CSC456H1 has 436 instead of 350
- [ ] CSC165: 
    Prerequisite: CSC108H1/CSC120H1 OR satisfy corequisite
    Corequisite: (CSC108H1/CSC120H, MAT137Y1/MAT157Y1)/ CSC148H1

### PR DESCRIPTION
**Pull request by david-yz-liu**
This is a new graph for the 2015-2016 year. I've moved a lot of things around, but I'm pretty happy with how the graph has turned out. A careful eye checking the prerequisites and some more testing would be appreciated. @christinem :)

@Ian-Stewart-Binks I made a minor modification to the SVG parser, but overall it worked quite well, even though I generated an entirely new Inkscape file from scratch.

### CODE REVIEW

**Comment by Ian-Stewart-Binks** [comment]
Cool! I'm temporarily on Windows, so I can't run the parser right now, but I'll make a few remarks on the new graph when I can. 

I feel as though grey may be too bland of a colour for math, and it's sort of kills the vibrancy of the graph. Perhaps [\#6276a3](http://www.color-hex.com/color/6276a3) could go nicely with it.

File: hs/SvgParsing/Generator.hs [hunk 1]
```diff
@@ -39,12 +39,12 @@ makeSVGDoc :: M.Map String String
            -> Bool    -- ^ Whether to include inline styles in the graph.
            -> S.Svg   -- ^ The completed SVG document.
 makeSVGDoc courseMap rects ellipses edges regions regionTexts styled =
-    S.docTypeSvg ! A.width "1052.3622"
-                 ! A.height "744.09448"
+    S.docTypeSvg ! A.width "1195"
```
Comment (Ian-Stewart-Binks) [inline hs/SvgParsing/Generator.hs]: Mmm. In the future, this should be changed to be based off of the width/height of input graph.

File: hs/Database/JsonParser.hs [hunk 2]
```diff
@@ -26,7 +26,8 @@ fbdbStr :: T.Text
 fbdbStr = "fdatabase1.sqlite3"
 
 dbStr :: T.Text
-dbStr = "Database/database2015.sqlite3"
+--dbStr = "Database/database2015.sqlite3"
+dbStr = "Database/database2015test.sqlite3"
```
Comment (Ian-Stewart-Binks) [inline hs/Database/JsonParser.hs]: We should choose one of these names.

**Comment by christinem** [comment]
@david-yz-liu I like the look of the new graph! 

Did a double check on all the prerequisites, they seem to be in order. :) Only one that needs to be fixed is CSC428: its prerequisite should actually be Sta2, not Sta1.

Did some testing and everything else seems to be working properly!

File: hs/SvgParsing/Generator.hs (hunk 1 above)
Comment (david-yz-liu) [inline hs/SvgParsing/Generator.hs]: Yes, definitely.

**Comment by david-yz-liu** [comment]
Thanks, @christinem

**Pull request merged**